openfl/experimental/workflow/component/aggregator/aggregator.py

```diff
@@ -11,10 +11,10 @@
 from threading import Event
 from typing import Any, Callable, Dict, List, Tuple
 
-from openfl.experimental.interface import FLSpec
-from openfl.experimental.runtime import FederatedRuntime
-from openfl.experimental.utilities import aggregator_to_collaborator, checkpoint
-from openfl.experimental.utilities.metaflow_utils import MetaflowInterface
+from openfl.experimental.workflow.interface import FLSpec
+from openfl.experimental.workflow.runtime import FederatedRuntime
+from openfl.experimental.workflow.utilities import aggregator_to_collaborator, checkpoint
+from openfl.experimental.workflow.utilities.metaflow_utils import MetaflowInterface
 
 
 class Aggregator:
```

Comment:
@teoparvanov
1. Basis our discussion with @psfoley and @kta-intel the changes to `openfl-tutorials/experimental/workflow/LocalRuntime` have been reverted as there is a different suggestion to refactor and reorganize the tutorials. This shall be covered in a separate PR and is aligned to your original comment [https://github.com/securefederatedai/openfl/pull/1169#issuecomment-2491017331](url)

2. Deprecating the "Aggregator-based-workflow" needs further discussion & alignment as it may be needed for replacing the task-runner API